data/Dels/Dels missions.txt

```diff
@@ -698,6 +698,214 @@ mission "First Contact: Dels Ufulre"
+#		fine 10000
+#		dialog `Officer <first>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+#
+#	npc evade
+#		government "Dels Ufulre"
+#		personality
+#			confusion 30
+#			plunders harvests target # no staying so they can flee
+#		fleet "Dels Ufulre Raid" 2
+#		system destination
+#
+#	npc
+#		government "Dels Kaote"
+#		personaility
+#			confusion 10
+#			heroic opportunistic frugal staying
+#		fleet "Dels Kaote"
+#		system destination
+#	on complete
+#		payment 100000
+#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`
+
+mission "Dels Officer: Patrol Systems"
+	landing
+	repeat
+	priority
+	name `Patrol systems`
+	source Delro
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	waypoint
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	description `You must patrol <stopovers> to check for Ufulre activity then return to <origin> to report.`
+
+	to offer
+		has "dels officer: report mission available"
+		has "Dels Officer: Patrol Mission" # In testing, I had a stack of these but it only offered once.
+	on offer
+		clear "dels officer: report mission available"
+		clear "Dels Officer: Patrol Mission"
+		conversation
+			`One you have landed, a Dels commander comes up and says, "Officer <first>, you must patrol <waypoints> to check on Ufulre activity, then return to <origin>."`
+			`	You get back to your ship and prepare for the mission.`
+
+				accept
+	on abort
+		"dels officer: bad officer" ++
+	on complete
+		payment 50000
+		dialog `You receive a message reading, "Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission."`
+
+mission "Dels Officer: Escort Convoy"
+	landing
+	repeat
+	priority
+	name `Escort convoy`
+	source Delro
+	destination
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
+	to offer
+		has "dels officer: report mission available"
+		has "Dels Officer: Escort Mission"
+	on offer
+		clear "dels officer: report mission available"
+		clear "Dels Officer: Escort Mission"
+		conversation
+			`Once you have landed, a Dels commander comes up to you and says, "Officer <first>, you must escort a small convoy of three Manoons to <destination> in order to help the people there recover from a recent raid. The ships are in the military priority docking."`
+			`	You get back to your ship and prepare for the mission.`
+				accept
+	on abort
+		"dels officer: bad officer" ++
+
+	npc
+		government "Dels Ufulre"
+		to spawn
+			"random" < 50
+		personality
+			confusion 30
+			plunders harvests
+		fleet "Dels Ufulre Patrol"
+		system
+			near Alkurud 0 1
+
+	npc
+		government "Dels Ufulre"
+		to spawn
+			"random" < 20

```

Comment:
```suggestion
		to spawn
			"random" < 20
			"armament deterrence" > 12

```
Since these fleets have chance to spawn Dacar heavy warship maybe lock one of the spawn behind player's fleet strength first so they have less chance of dealing with 2 Dacar at the same time.
Ideally I probably should make the Ufulre fleet more granular and balance out the strength of each variant but that'll be later.